oracle-plugin/widgets/Oracle-connector.json

```diff
@@ -118,5 +130,6 @@
       ]
     }
   ],
+  "Filters" : [],
```

Comment:
Instead of empty, we need to add a similar filter for isolation levels as in source and sink.

Reply:
I added the filter but I don't see the ISOLATION LEVEL showing up.

Reply:
yeah looks like there is some issue in the UI not rendering it correctly? Is it only with `select` widget? Did you try changing the widget type?

Reply:
I was able to verify your changes, looks like it is a UI bug when there are 2 connectors, one in system scope and other in user scope, UI only renders system scope one.

Reply:
I tired the radio button but it didn't render. I thought the filter capitalization error would fix it but no dice.

Reply:
Opened a JIRA for UI team - https://cdap.atlassian.net/browse/PLUGIN-1248